### PR TITLE
Add bip39 support

### PR DESCRIPTION
**Pull request by romaniy173**
Issue: add bip39 support
now should be possible to import wallet by mnemonic sentence 

This one is depends on : https://github.com/BlockSettle/common/pull/1940

### CODE REVIEW

File: BlockSettleSigner/qml/BsDialogs/WalletImportDialog.qml [hunk 1]
```diff
@@ -207,6 +263,18 @@ CustomTitleDialogWindow {
                                 }
                             }
                         }
+
+                        BSPib39Input {
```
Comment (sergey-chernikov) [inline BlockSettleSigner/qml/BsDialogs/WalletImportDialog.qml:267]: Pib?

File: BlockSettleSigner/qml/WalletsPage.qml [hunk 2]
```diff
@@ -124,7 +123,12 @@ Item {
                         enabled: buttonRow.enableButtons
                         text: qsTr("Delete")
                         onClicked: {
-                            JsHelper.deleteWalletDialog(getCurrentWalletIdData())
+                            let dialog = JsHelper.deleteWalletDialog(getCurrentWalletIdData())
+                            dialog.bsAccepted.connect(function(){
+                                console.log("HERE", walletsView_.selection.hasSelection
```
Comment (sergey-chernikov) [inline BlockSettleSigner/qml/WalletsPage.qml:128]: This can now be removed, I guess?

File: BlockSettleSigner/signer.qrc [hunk 3]
```diff
@@ -81,5 +81,16 @@
...
+        <file>bip39Dictionaries/chinese_simplified.txt</file>
+        <file>bip39Dictionaries/chinese_traditional.txt</file>
+        <file>bip39Dictionaries/czech.txt</file>
+        <file>bip39Dictionaries/english.txt</file>
+        <file>bip39Dictionaries/french.txt</file>
+        <file>bip39Dictionaries/italian.txt</file>
+        <file>bip39Dictionaries/japanese.txt</file>
+        <file>bip39Dictionaries/korean.txt</file>
+        <file>bip39Dictionaries/spanish.txt</file>
+        <file>qml/BsControls/BSPib39Input.qml</file>
```
Comment (sergey-chernikov) [inline BlockSettleSigner/signer.qrc:93]: Let's not mix dictionaries and qml files - at least don't put them interleaved.

File: BlockSettleSigner/qml/BsDialogs/WalletImportDialog.qml (hunk 1 above)
Comment (romaniy173) [reply BlockSettleSigner/qml/BsDialogs/WalletImportDialog.qml]: typo, fixed, thank you

File: BlockSettleSigner/qml/WalletsPage.qml (hunk 2 above)
Comment (romaniy173) [reply BlockSettleSigner/qml/WalletsPage.qml]: Fixed, thanks

File: BlockSettleSigner/signer.qrc (hunk 3 above)
Comment (romaniy173) [reply BlockSettleSigner/signer.qrc]: Added a different qrc file for storing dictionaries there, thank you

**Pull request merged**